docs/source/configuration/multilingual.md

```diff
@@ -28,22 +29,37 @@ config.settings = {
 }
 ```
 
-Declare the language you want to support in your site in the `supportedLanguages` array,
-and which is the default language of your site.
+Declare the language you want to support in your site in the `supportedLanguages` array, and which is the default language of your site.
 
 ```{warning}
-The default language and the supported languages must match the one set in the Plone
-side, and those should be set using GenericSetup using your policy package, or
-manually via the Languages control panel, i.e. en for English, or pt-br for Portuguese (Brazil)
+The default language and the supported languages must match those set in the Plone backend.
+Those should be set either using GenericSetup using your policy package or manually via the {guilabel}`Languages` control panel.
+Examples include `en` for English, or `pt-br` for Portuguese (Brazil).
+```
+
+## Docker backend configuration
+
+If you use the Docker image [`plone-backend`](https://github.com/plone/plone-backend), you can set its `LANGUAGE` environment variable, overriding the default of `en`, when you start it.
+
+You can either pass an environment variable into the make command to start the backend, or export an environment variable in your shell session and start the backend.
+
+```shell
+# pass method
+make backend-docker-start LANGUAGE=pt-br
```

Comment:
I tried this command but the site was still created in English.

Reply:
LANGUAGE needs to be set as an environment variable in the shell, not passed to make as an argument:

```suggestion
LANGUAGE=pt-br make backend-docker-start
```

Reply:
It works both ways.